### PR TITLE
Show an ex_perimental docker images section

### PR DESCRIPTION
**Pull request by kritika-singh3**
Issue: #1136 

Added an experimental docker images section which points to `gocdexperimental/{image_name}`

P.S. - the weird title is go around the mergeable check

### CODE REVIEW

**Comment by kritika-singh3** [comment]
Preview: https://gocd.github.io/pr-review.gocd.org/pr-1148/download/#docker

File: source/assets/javascripts/app/_download.js [hunk 1]
```diff
@@ -158,6 +160,10 @@ var showDownloadLinks = (function ($) {
         $("#download-revisions-template").html()
       );
 
+      // adding the docker organisation
+      latest_cloud_release.docker_org = settings.docker_org;
+      other_cloud_releases.docker_org = settings.docker_org;
```
Comment (arvindsv) [inline source/assets/javascripts/app/_download.js:165]: Instead of this, you should use `R.assoc`. See `addDisplayVersion` and `addReleaseDate` functions.

Comment (kritika-singh3) [reply source/assets/javascripts/app/_download.js]: Done.

**Comment by arvindsv** [comment]
@kritika-singh3 This looks better.

There's one small issue. At the bottom of the page, after you click "Show old releases", this shows up:

**Comment by arvindsv** [comment]
Also note that the org does not show up, in old releases.

**Comment by kritika-singh3** [comment]
Fixed the issue with no org. The `docker_org` was getting associated with the array instead of object.

**Comment by arvindsv** [comment]
That works. However, it shows some images which don't seem to be present on Docker Hub. Like this one:



I suppose that's a problem with the cloud.json not having been updated?

**Comment by arvindsv** [comment]
Apart from that, the change looks good to me. Mergable.

**Comment by kritika-singh3** [comment]
> I suppose that's a problem with the cloud.json not having been updated?

@arvindsv You are right. This is because we always append to the cloud json rather than recreating it. Will fix it, verify and then merge this PR. :)

**Pull request merged**